tests/ui_tests/app/page/cluster_list/page.py

```diff
@@ -172,6 +178,7 @@ def run_upgrade_in_cluster_row(
         upgrade_name: str,
         config: Optional[dict] = None,
         hc_acl: bool = False,
+        is_new_host: bool = False,
```

Comment:
please describe this parameter's purpose in docstring:
1. What it will do (create new host in complex upgrade popup, I guess)
2. Interaction with other params (it is dependant on `hc_acl` and we need to mention that there's no point of passing `is_new_host` without `hc_acl=True`

Reply:
added